.gitignore

```diff
@@ -41,3 +41,6 @@ Thumbs.db
 # Certification Files
 cert.pem
 key.pem
+
+.scannerwork
+Mull
```

Comment:
Just curious, what is in the Mull file/folder?

Reply:
I opened up a folder called Mull to put the project in, but when I pulled the project, it created another folder called Mull (the actual project folder), so I put the first folder in gitignore and forgot to take it out. I'll take it out next time I do a commit to avoid force pushing.